### DISCUSSION TITLE
Tests

### DISCUSSION THREAD

**Discussion by hkamran80** [FujoWebDev/AO3.js#59] [General]
Why is CircleCI being used for tests, as opposed to something like GitHub Actions?

**Comment by essential-randomness** [comment]
Just legacy reasons. I had set it up like this in the very first project I
made years ago, and then just kept using it. If you do want to swap it out
for GH actions, I'm really not that attached to it being CircleCI.

**Reply by hkamran80** [reply]
Ah, gotcha. Thanks for explaining!
